src/Representation/CollectionRepresentation.php

```diff
@@ -7,15 +7,9 @@
 use Hateoas\Configuration\Annotation as Hateoas;
 use JMS\Serializer\Annotation as Serializer;
 
-/**
- * @Serializer\ExclusionPolicy("all")
```

Comment:
hmm., this change makes it mandatory to have the attribute driver, right? is this a BC break?

Reply:
Yes, I think so.

I just added the annotations again to avoid that possible BC break.